app/models/institution_builder.rb

```diff
@@ -448,4 +449,36 @@ def self.add_school_closure(version_number)
 
     Institution.connection.update(str)
   end
+
+  def self.build_zip_code_rates_from_weams(version_number)
+    timestamp = Time.now.utc.to_s(:db)
+    conn = ActiveRecord::Base.connection
+
+    str = <<-SQL
+      INSERT INTO zipcode_rates (
+        zip_code,
+        mha_rate_grandfathered,
+        mha_rate,
```

Comment:
What about the other `ZipcodeRate` attributes?
`mha_name`
`mha_code`
I was searching around the `vets-website` codebase, and I'm not sure that `mha_code` is used anywhere, but `mha_name` definitely _is_ used.

Reply:
They're not currently in the WEAMS data, and we have approval from the VA to remove `mha_name` from vets-website.

Reply:
Are they going to be added to the WEAMS data?  If not, should we drop those columns from the table, and remove references to them from the codebase (e.g. `ZipcodeRateSerializer`)?

Reply:
Unknown.  We're going to hold off making any changes until later because getting this in as-is is very high priority for them.